server.js

```diff
@@ -0,0 +1,19 @@
+const http = require('http');
+
+const hostname = '0.0.0.0';
+const port = 3000;
+
+// Computes fibonacci:
+let a = 1;
+let b = 1;
+for (let i = 0; i < 10; i += 1) [a, b] = [b, a + b];
```

Comment:
Incorrect Fibonacci Computation

The right version:
let a = 0;
let b = 1;
for (let i = 0; i < 10; i += 1) {
  [a, b] = [b, a + b];
}